.github/workflows/go.yml

```diff
@@ -46,3 +46,8 @@ jobs:
     - name: Validate the examples
       run: make install validate-examples
       id: validate-examples
+
+    - name: Commit generated files
+      uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
We're using a different action on other workflows: https://github.com/elastic/terraform-provider-ec/blob/master/.github/workflows/branch.yml#L44-L51. Any reason not to re-use the same one here?

Reply:
Thank you for the suggestion, Marc 👍 

> Any reason not to re-use the same one here?

Not at all, I just missed it. I will change my PR to use this one

Reply:
Fixed

Reply:
> We're using a different action on other workflows: https://github.com/elastic/terraform-provider-ec/blob/master/.github/workflows/branch.yml#L44-L51. 

I switched to it, but having an issue similar to https://github.com/EndBug/add-and-commit/issues/199. Trying to fix it...